openedx/core/djangoapps/course_groups/migrations/0002_change_inline_default_cohort_value.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('course_groups', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='coursecohortssettings',
+            name='always_cohort_inline_discussions',
+            field=models.BooleanField(default=False),
```

Comment:
So, the good new is that this is a tiny table, so even modifying it should be fine (3k rows).
InnoDB should be able to do this as purely a metadata change https://dev.mysql.com/doc/refman/5.6/en/innodb-create-index-overview.html which is even faster.

I'm playing with something locally, but suspect this is fine to go without my poking being complete.